workflow-dev/tpl/deis-workflow-manager-rc.yaml

```diff
@@ -25,19 +25,19 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        - name: WORKFLOW_MANAGER_PORT
+        - name: PORT
           value: "8080"
-        - name: WORKFLOW_MANAGER_VERSIONSAPIURL
+        - name: VERSIONS_API_URL
           value: {{.workflowManager.versionsApiURL}}
-        - name: WORKFLOW_MANAGER_DOCTORAPIURL
+        - name: DOCTOR_API_URL
```

Comment:
cool, thanks @arschles ; fyi created https://github.com/deis/deisrel/pull/63 to track the deisrel addition